src/main/java/net/mcreator/element/parts/gui/GUIComponent.java

```diff
@@ -77,6 +78,11 @@
 
 	public abstract int getHeight(Workspace workspace);
 
+	/**
+	 * Returns the priority for when this component should be drawn in the UI, to represent how Minecraft draws components in the game.
+	 *
+	 * @return The priority of the component (lower is more important, higher is less important)

```

Comment:
> lower is more important, higher is less important

This is not correct. Lower means it will be rendered closer to the screen and higher means it will "sink" more behind other components